### PR TITLE
Add type to YAML main for Builds

### PR DESCRIPTION
**Pull request by nebelsprachen**
Update binaries and YAML

Version(s):

cherrypick `build-docs-1.1`

Issue:

Link to docs preview:

https://79750--ocpdocs-pr.netlify.app/openshift-builds/latest/work_with_builds/using-builds.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/ob-creating-a-s2i-build.adoc [hunk 1]
```diff
@@ -39,7 +40,7 @@ spec:
   - name: builder-image
     value: quay.io/centos7/nodejs-12-centos7
```
Comment (apoorvajagtap) [inline modules/ob-creating-a-s2i-build.adoc:41]:
```suggestion
    value: quay.io/centos7/nodejs-12-centos7:master
```
This image needs the tag.

File: modules/ob-creating-a-s2i-build.adoc [hunk 2]
```diff
@@ -39,7 +40,7 @@ spec:
...
   output:
-    image: quay.io/<repo>/s2i-nodejs-example <4>
+    image: quay.io/centos7/nodejs-12-centos7:master <4>
```
Comment (apoorvajagtap) [inline modules/ob-creating-a-s2i-build.adoc:43]:
```suggestion
```
- This needs to be kept as it. User is expected to specify the repo as per their need.

**Review by apoorvajagtap — CHANGES_REQUESTED** [review]

**Review by apoorvajagtap — APPROVED** [review]

**Review by xenolinux — COMMENTED** [review]
LGTM

Gentle reminder to squash your commits. 

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

**Pull request merged**